importer/main.py

```diff
@@ -302,6 +303,24 @@ def location_extras(resource, payload_string):
         del obj["resource"]["type"]
         payload_string = json.dumps(obj, indent=4)
 
+    try:
+        if len(adminLevel) > 0:
+            payload_string = payload_string.replace("$adminLevelcode", adminLevel)
+        else:
+            base_url = get_base_url()
+            check_admin_level_url = "/".join([base_url, "Location", parentID])
+            response = handle_request("GET", "", check_admin_level_url)
+            response_obj = json.loads(response[0])
+            parent_admin_level = response_obj["type"][1]["adminLevel"][0]["code"]
+            admin_level = parent_admin_level + 1
+            obj = json.loads(payload_string)
+            obj["resource"]["type"][1]["adminLevel"][0]["code"] = admin_level
+            obj["resource"]["type"][1]["adminLevel"][0]["display"] = admin_level
+            payload_string = json.dumps(response_obj, indent=4)
+    except KeyError:
+        logging.info("This is a root location, admin level will be 0")
```

Comment:
We probably need to add an extra check here, because there are some cases where the admin levels has just not been added to the json, so it's not necessarily that this is the root